metron-analytics/metron-profiler/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,152 @@
+
+metron-profiler
+Copyright 2006-2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project contains annotations derived from JCIP-ANNOTATIONS
+Copyright (c) 2005 Brian Goetz and Tim Peierls. See http://www.jcip.net
+
+===============================================================================
+
+The BracketFinder (package org.apache.commons.math3.optimization.univariate)
```

Comment:
Double check that you are actually bundling the packages which are listed below. If you aren't, you can remove the corresponding section.